x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list_logic.ts

```diff
@@ -73,7 +114,29 @@ export const EnginesListLogic = kea<MakeLogicType<EngineListValues, EnginesListA
     ],
   }),
   selectors: ({ selectors }) => ({
-    meta: [() => [selectors.parameters], (parameters) => parameters.meta],
+    deleteModalEngineName: [() => [selectors.deleteModalEngine], (engine) => engine?.name ?? ''],
+    isDeleteLoading: [
+      () => [selectors.deleteStatus],
+      (status: EngineListValues['deleteStatus']) => [Status.LOADING].includes(status),
+    ],
+    isLoading: [
+      () => [selectors.status],
+      (status: EngineListValues['status']) => [Status.LOADING].includes(status),
+    ],
     results: [() => [selectors.data], (data) => data?.results ?? []],
+    meta: [() => [selectors.parameters], (parameters) => parameters.meta],
+  }),
+  listeners: ({ actions, values }) => ({
+    apiError: (e) => flashAPIErrors(e),
```

Comment:
suggestion: I think you can delete these flashApiErrors listeners, that should be happening automatically now.